web-client/public/gene/info-wip.css

```diff
@@ -0,0 +1,172 @@
+
```

Comment:
Please check if this file is actually being used by the code base—the filename makes me wonder. `wip` typically means "work in progress," and if this file was just that, with the final work actually going to _info.css_, then we don’t need to keep this one.

Reply:
It is not, it is deleted